src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type TodoProps = {
+  todos: Todo[];
+  selectedTodoId: number;
+  onSelectTodo: (id: number) => void;
+};
+
+export const TodoList: React.FC = ({
```

Comment:
The `TodoList` component is defined as a `React.FC` but does not specify the props type. You should define it as `React.FC<TodoProps>` to ensure type safety and proper type checking.